docs/docs/write-pages.md

```diff
@@ -59,70 +59,71 @@ The dynamic files that are created are (all under the `.cache` directory).
 
 ## pages.json
 
-This is a collection of page objects, created from redux `pages` namespace. For each page it includes the
+Esta é uma coleção de objetos de página, criados a partir do namespace redux `pages`. Para cada página, inclui o
 
-- [componentChunkName](/docs/behind-the-scenes-terminology/#componentchunkname)
-- [jsonName](/docs/behind-the-scenes-terminology/#jsonname)
-- [path](/docs/behind-the-scenes-terminology/#path)
-- [matchPath](/docs/behind-the-scenes-terminology/#matchpath)
+- [componentePedacoNome](/docs/behind-the-scenes-terminology/#componentchunkname)
+- [jsonNome](/docs/behind-the-scenes-terminology/#jsonname)
+- [caminho](/docs/behind-the-scenes-terminology/#path)
+- [combineCaminho](/docs/behind-the-scenes-terminology/#matchpath)
```

Comment:
```suggestion
- [componentChunkName](/docs/behind-the-scenes-terminology/#componentchunkname)
- [jsonName](/docs/behind-the-scenes-terminology/#jsonname)
- [path](/docs/behind-the-scenes-terminology/#path)
- [matchPath](/docs/behind-the-scenes-terminology/#matchpath)
```

Reply:
Creio que neste caso é melhor manter o nome original pois quando o usuário for utilizar a ferramenta ele encontrará esses nomes [dentro do código](https://www.gatsbyjs.org/docs/gatsby-internals-terminology/#redux-components-namespace).